docs/extend/creating-classic-plugins.md

```diff
@@ -137,8 +137,8 @@ Allows code to access the filesystem, to read or write paths as specified by the
 
 It is possible to specify 3 different types of file entitlement:
```

Comment:
@ldematte final thought on this one: we could use numbered list for the 3 here to be even clearer :)